api/maven-api-core/src/main/java/org/apache/maven/api/Constants.java

```diff
@@ -462,6 +462,15 @@ public final class Constants {
     @Config(type = "java.lang.Boolean", defaultValue = "true")
     public static final String MAVEN_CONSUMER_POM = "maven.consumer.pom";
 
+    /**
+     * User property for controlling "maven personality". If set to {@code true} Maven will behave
+     * as previous major version, Maven 3.
+     *
+     * @since 4.0.0
+     */
+    @Config(type = "java.lang.Boolean", defaultValue = "false")
```

Comment:
MAVEN_MAVEN3_PERSONALITY sounds weird and duplicative. Maybe just MAVEN3_PERSONALITY?